php_phongo_classes.h

```diff
@@ -130,7 +130,6 @@
 typedef struct {
 	zend_object_iterator   intern;
 	php_phongo_cursor_t   *cursor;
-	long                   current;
```

Comment:
This was relocated to our cursor struct so we don't need a bi-directional reference between `php_phongo_cursor_iterator` and `php_phongo_cursor_t`. Also, the current document is already stored there (in `php_phongo_bson_state visitor_data`).